website/docs/advanced-concepts/access-control/roles.md

```diff
@@ -0,0 +1,247 @@
+
+
+# Roles
+ A Role is a collection of permissions that grant access to perform certain operations on resources and can be assigned to a user or a group. Roles enable users to manage multiple permissions rather than assigning individual permissions to users/groups. In Appsmith, a role can be mapped to multiple permissions and a permission can be mapped to multiple roles. However, a role cannot be nested, i.e. it can contain only permissions and not other roles. Roles are useful for efficiently managing permissions to access Appsmith resources.
+ 
+ Appsmith provides three built-in roles - **Admin**, **Developer** and **App Viewer**.  With GAC, you can create a custom role that provides fine-grained access control. 
+
+## Resources
+
+A resource refers to specific components or aspects of an application that can be accessed or modified by users. These resources may include individual pages or views within the application, specific data sets or database tables, or specific actions or functions that can be performed within the application. In Appsmith, resources are classified into the following categories:
+
+* **App Resources:** These are resources related to the applications built in Appsmith, such as individual pages or views within the application.
+* **Datasource and Queries:** These are resources related to databases or queries created in Appsmith, such as a MongoDB setup used as a data source.
+* **Groups and Roles:** These are resources related to the roles available for the workspaces in an Appsmith instance . For example, if you have two workspaces - HR and Finance - then roles related to HR and finance would be available.
+* **Others:** These are resources related to workspaces and audit logs in Appsmith.
```

Comment:
Should we mention that the remaining resources come here instead of mentioning workspaces and audit logs? Asking because if we add new things on this page in future then we would also need update on the docs.

Reply:
I think for now we should mention what's there in others. When we add more things, we'll changes it in the docs.